src/components/common/dashboard/OperationComponent.vue

```diff
@@ -0,0 +1,390 @@
+<template>
+    <div
+        :id="'op_' + shownOpId"
+        class="flex flex-col shadow-xl bg-white hover:bg-gray-200 rounded-lg items-start p-2 sm:p-4"
+        @click="toggleDetails"
+    >
+        <div class="flex w-full items-center justify-between sm:justify-start mb-2">
+            <div class="w-full sm:flex sm:items-center">
+                <div class="flex items-center">
+                    <i
+                        v-if="showWatchOption"
+                        :id="'toggleWatch_op_' + shownOpId"
+                        class="text-md hover:text-blue-600 text-blue-500 far fa-bookmark cursor-pointer"
+                        :class="{ 'fas fa-bookmark': isWatched }"
+                        :title="isWatched ? 'Unwatch' : 'Watch'"
+                        @click.stop="toggleWatch"
+                    ></i>
+                    <div class="ml-1 text-xs font-semiboldtext-gray-600 uppercase">{{ type }}</div>
+                    <span
+                        :id="'op_state_' + shownOpId"
+                        class="ml-4 inline-block px-2 text-xs font-semibold tracking-wide text-teal-800 uppercase rounded-full"
+                        :class="statusColor"
+                    >
+                        {{ operation.state }}
+                    </span>
+                </div>
+                <div class="flex items-center sm:ml-6">
+                    <div v-if="!isRejected" class="flex mr-1" :title="'Approvals (' + missingApprovals + ' missing)'">
+                        <i v-for="index in approvals" :key="index" class="text-sm text-green-400 far fa-check-circle"></i>
+                        <i v-for="index in missingApprovals" :key="index" class="text-sm text-green-400 far fa-circle"></i>
+                    </div>
```

Comment:
Do we know something about the approvals/ missing approvals?
It would be nice to see what is missing and who already approved (nice to have ;) )

Reply:
Approvals and missing approvals is a very ugly thing for us in the frontend. The two arrays within the operations object contain groups and users for both, but the way how approvals are given is somewhat hard to display, as a user signs an operation for himself AND his group. 
The component above was just meant to display some sort of progress. Additionally, in the object, just enrollmentIds are given due to gdpr, so we are not allowed to know anything relevant about a missing approval of an user as an average user.